### PR TITLE
Re-upload dataset to hub to avoid using script upload

### PR DESCRIPTION
**Pull request by imenelydiaker**
This should solve issue #1311.
I re-uploaded the dataset to parquet format here: https://huggingface.co/datasets/mteb/mlsum as suggested by @lhoestq, to avoid using the script generation for loading.

### CODE REVIEW

File: mteb/benchmarks/benchmarks.py [hunk 1]
```diff
@@ -63,6 +63,7 @@ def load_results(
             base_results = load_results()
         return base_results.select_tasks(self.tasks)
 
+
```
Comment (isaac-chung) [inline mteb/benchmarks/benchmarks.py:66]: nit: no-op, maybe can remove?

**Review by isaac-chung — APPROVED** [review]
Looks good, as long as you've tried out at least one model on the new dataset.

File: mteb/benchmarks/benchmarks.py (hunk 1 above)
Comment (imenelydiaker) [reply mteb/benchmarks/benchmarks.py:66]: It's the linter 😅

**Comment by imenelydiaker** [comment]
> Looks good, as long as you've tried out at least one model on the new dataset.

Yes I tested it locally with average_kominos model. Let's merge?

**Comment by KennethEnevoldsen** [comment]
Great fix, @imenelydiaker!

**Pull request merged**